v0x04/flow.py

```diff
@@ -73,12 +73,12 @@ def as_of_action(self):
 class ActionPopVlan(ActionBase):
     """Action to pop the outermost VLAN tag."""
 
-    def __init__(self, *args):
+    def __init__(self):
         """Initialize the action with the correct action_type."""
         self.action_type = 'pop_vlan'
 
     @classmethod
-    def from_of_action(cls, of_action):
+    def from_of_action(cls):
```

Comment:
It's not good to change a method signature unless you are 100% sure you should do it.